mempool.go

```diff
@@ -1103,9 +1110,10 @@ func (mp *txMemPool) LastUpdated() time.Time {
 
 // newTxMemPool returns a new memory pool for validating and storing standalone
 // transactions until they are mined into a block.
-func newTxMemPool(cfg *mempoolConfig) *txMemPool {
+func newTxMemPool(policy *mempoolPolicy, cfg *mempoolConfig) *txMemPool {
```

Comment:
The `mempoolConfig` struct has the `Policy` field in it.  There shouldn't be a need to pass them both.